cnxpublishing/db.py

```diff
@@ -35,8 +35,11 @@
     ResourceFileExceededLimitError,
     UserFetchError,
     )
-from .utils import parse_archive_uri, parse_user_uri
+from .models import inject_mathml_svgs
 from .publish import publish_model, republish_binders
+from .utils import (
+    parse_archive_uri, parse_user_uri,
+    )
```

Comment:
They aren't used in this diff, but are used in the module. It was changed to a multiline import because at one point I was going to put something in utils.